packages/client/hmi-client/src/components/documents/tera-text-editor.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+	<QuillEditor theme="snow" v-model:content="text" contentType="html" />
+</template>
+
+<script setup lang="ts">
+import { ref, watch } from 'vue';
+import { QuillEditor } from '@vueup/vue-quill';
+import '@vueup/vue-quill/dist/vue-quill.snow.css';
+
+const props = defineProps<{
+	initialText: string;
+}>();
```

Comment:
No you misunderstood. I am saying why ware we calling it `initialText`, it gives the impression that the text is going to go under some kind of transformation.

Reply:
Ah. Open to suggestions for a different name, if I could I would name the prop and the ref text. `initialText` makes sense to me as the "initial value of `text`", which is true. And technically `text` could be transformed.

Reply:
In fact `text` *is* transformed as user can change the content displayed in the editor (though nothing is being saved currently)

Reply:
Erm... how about `textContent`? 

If we are allowed to changed the text, then we need to be careful here because changing props is generally an anti-pattern in Vue, as it can have unforeseen impact upstream if the ancestor components are also depending on the same field. I lack the full context here so maybe @YohannParis and yourself can decide the best route to follow.

Reply:
`textContent` sounds good

We won't be changing the props since even though `text` is a ref of `prop.text`, changes to `text` will not change `prop.text` and any upstream components since `prop.text` is a primitive type.

Reply:
Okay thanks - I think I misread the props handling code. Let's just make the name change and merge 👍